homework3.py

```diff
@@ -0,0 +1,86 @@
+import functools
+
+
+def divisor_of_one_hundred(func):
+    @functools.wraps(func)
+    def inner(*args, **kwargs):
+        res = func(*args, **kwargs)
+        if 100 % res == 0:
+            print("We are OK!")
+        else:
+            print(f"Bad news guys, we got {100 % res}")
+        return res
+    return inner
+
+
+def not_str(func):
+    @functools.wraps(func)
+    def inner(*args, **kwargs):
+        if True in [isinstance(el, str) for el in (args + tuple(kwargs.values()))]:
```

Comment:
check the `any()` or `all()` functions in Python, they will bring you more clear code